articles/getting-started/requirements-for-sending-domains.md

```diff
@@ -13,29 +13,38 @@ The gold standard for a **Responsible Sender** is one who sends only to those re
 * Stops sending to recipients who have unsubscribed
 * Has a clear web presence, usually where a customer can sign up and manage their experience
 
-As part of being a **Responsible Sender,** we ask all customers to complete the sending domain validation process -- after which your domain(s) will be activated and ready for sending.  This process has two steps:
+As part of being a **Responsible Sender,** we ask all customers to complete the domain validation process -- after which your domain(s) will be activated and ready for use.  This process has two steps:
 
-1. Ownership Verification (DKIM or Abuse/Postmaster setup)
-1. Compliance Review
+### Step One: Ownership Verification
+
+First, we need to make sure you own the domain you want to set up. *Adding a DNS record* is the recommended way to show that you own your domain. Depending on how you plan to use the domain, you'll want to add different DNS records so you can verify ownership and get the domain properly set up all at once. [Learn why you should set up both a sending and a bounce domain]().
```

Comment:
Okay, I'll hold on doing the pull request until you decide on if you want to link or remove. Please update this comment accordingly with final disposition.